person.rb

```diff
@@ -0,0 +1,38 @@
+# Create a class person
```

Comment:
- [ ] :name and :age instance variable should be Getters and Setters are defined
  attr_accessor :name, :age 

- [ ]  :id should only be able to be retrieved so only the Getter is defined

  attr_reader :id